wpilibjExamples/src/main/java/edu/wpi/first/wpilibj/examples/statespacedifferentialdrivesimulation/RobotContainer.java

```diff
@@ -24,9 +25,12 @@
 import edu.wpi.first.wpilibj2.command.RunCommand;
 import edu.wpi.first.wpilibj2.command.button.JoystickButton;
 
-import java.util.List;
+import edu.wpi.first.wpilibj.examples.statespacedifferentialdrivesimulation.subsystems.DriveSubsystem;
 
 import static edu.wpi.first.wpilibj.XboxController.Button;
+import static edu.wpi.first.wpilibj.examples.statespacedifferentialdrivesimulation.Constants.AutoConstants;
+import static edu.wpi.first.wpilibj.examples.statespacedifferentialdrivesimulation.Constants.DriveConstants;
+import static edu.wpi.first.wpilibj.examples.statespacedifferentialdrivesimulation.Constants.OIConstants;
```

Comment:
Static imports are considered bad style.  Why was this changed?

Reply:
IDK why wpiformat/checkstyle ignore those imports. However, I think those classes can be imported non-statically, I'll change to that.

Reply:
Yes, please